circuits/transfer/src/execute/input.rs

```diff
@@ -92,23 +91,9 @@ impl<T, const H: usize, const A: usize> CircuitInput<T, H, A> {
         let value_commitment = note.value_commitment();
         let value_commitment = composer.append_point(value_commitment);
 
-        let nonce = hash_inputs[3];
-        let nonce = composer.append_witness(nonce);
-
-        let r = note.stealth_address().R();
-        let r = composer.append_point(r);
-
-        let pos = hash_inputs[8];
+        let pos = hash_inputs[5];
```

Comment:
Just a question: is this removal documented somewhere? Curious to know why

Reply:
It came from this issue [https://github.com/dusk-network/phoenix-core/issues/123]

Reply:
I'll add it to the commit too